crates/holochain/src/conductor/p2p_store.rs

```diff
@@ -207,6 +211,106 @@ pub fn exchange_peer_info(envs: Vec<EnvironmentWrite>) {
     }
 }
 
+/// Get agent info for a single agent
+pub fn get_agent_info_signed(
+    environ: EnvironmentWrite,
```

Comment:
This pattern of just using a raw environment to get stuff out of the database is unprecedented, but it's fine especially if we're doing database refactor soon